ci/actions/linux/deploy-docker.sh

```diff
@@ -8,7 +8,7 @@ tags=()
 if [ -n "$TRAVIS_TAG" ]; then
     tags+=("$TRAVIS_TAG")
     if [[ "$GITHUB_WORKFLOW" = "Beta" || "$GITHUB_WORKFLOW" = "Test" ]]; then
-        tags+=(latest latest-including-rc)
+        tags+=(latest)
```

Comment:
@argakiig If we only have `latest` here is this needed? I vaguely recall that tag being added by default, not needing it specifically. If that is the case, then the selective removal of `latest` for live in line 55 below should be enough.

Reply:
Good call, will adjust to just remove this case entirely